Makefile

```diff
@@ -171,7 +171,7 @@ analyze: $(JS)
 		echo "ccflags-y += -DZJS_SNAPSHOT_BUILD" >> src/Makefile; \
 	fi
 ifeq ($(DEV), ashell)
-	@cat fragments/prj.mdef.dev >> prj.mdef
+	@cat fragments/prj.mdef.ashell >> prj.mdef
```

Comment:
couldn't this be moved into the bottom section where you append the prj.conf? so you don't need a duplicate ifeq